### DISCUSSION TITLE
GhostRider algorithm not functioning when miner connects

### DISCUSSION THREAD

**Discussion by nmweaver** [coinfoundry/miningcore#983] [Q&A] [answered]
I've been trying to implement an algorithm called GhostRider for the currency Raptoreum. Although it detects the global stats and difficulty, it instantly gives this error whenever a miner connects:

`./Miningcore: symbol lookup error: /path/to/libmultihash.so: undefined symbol: _Z7gr_hashPKcPc`

Does anyone know why this is happening? The symbol is supposed to be `gr_hash`. The repository can be found [here](https://github.com/LOUD-Mining/miningcore). This cryptocurrency is a fork off of Dash and is a PoW coin.

**Comment by nmweaver** [comment, marked as answer]
I found the problem. I had accidentally forgotten to import the `gr.h` file properly.

**Reply by MotoAcidic** [reply]
Was testing this out today and everything but submitting shares seems to work from your repo. What is the config.json you are using?

**Reply by oliverw** [reply]
The share submission problem should be fixed now.

**Reply by oliverw** [reply]
Yes, and now would you stop posting stuff related to fork here? Thank you.
